plugins/source/alicloud/codegen/recipies/oss.go

```diff
@@ -0,0 +1,47 @@
+package recipies
+
+import (
+	"github.com/aliyun/aliyun-oss-go-sdk/oss"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func OSS() []*Resource {
+	bucketName := codegen.ColumnDefinition{
+		Name:     "name",
+		Type:     schema.TypeString,
+		Resolver: `client.ResolveParentColumn("Name")`,
```

Comment:
I think we should be able to use `schema.ParentColumnResolver` from the plugin-sdk library here (e.g. https://github.com/cloudquery/cloudquery/blob/ec1d132529824d16682a92a83ee326fcba939efc/plugins/source/aws/codegen/recipes/s3.go#LL61C2-L61C2), and remove the custom resolver from this plugin. I notice this pattern was copied from the Github plugin, we should probably fix it there too in a follow-up PR :)